qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AddEditNewPrivilegePage.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.page;
+
+import org.openmrs.contrib.qaframework.helper.Page;
+import org.openqa.selenium.By;
+
+public class AddEditNewPrivilegePage extends Page{
+
+	protected static final String PAGE_URL = "/adminui/metadata/privileges/privilege.page?action=add&";
```

Comment:
What happens when you make this url as  **protected static final String PAGE_URL = "/adminui/metadata/privileges/privilege.page";** ?

Reply:
Why not private?